streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamPartitionAssignor.java

```diff
@@ -361,8 +388,10 @@ public Subscription subscription(Set<String> topics) {
 
         // assign tasks to clients
         states = TaskAssignor.assign(states, partitionsForTask.keySet(), numStandbyReplicas);
-        Map<String, Assignment> assignment = new HashMap<>();
 
+        final List<AssignmentSupplier> assignmentSuppliers = new ArrayList<>();
+
+        final Map<HostInfo, Set<TopicPartition>> endPointMap = new HashMap<>();
```

Comment:
We don't enforce it though, and i don't think we want to. Some people wont care about QS, so why should they be forced to provide a host:port pair in their config?